utilities/cluster_config_utils.py

```diff
@@ -133,7 +133,10 @@ def is_cbs_ssl_enabled(cluster_config):
 def is_x509_auth(cluster_config):
     ''' Load cluster config to see if auth should be done using x509 certs '''
     cluster = load_cluster_config_json(cluster_config)
-    return cluster["environment"]["x509_certs"]
+    val = cluster["environment"]["x509_certs"]
+    if val == "True" or val == "true":
```

Comment:
you do not need to add condition here. On conftest file, when it ready pytest parameters, it should assign false as default. Like enabling xattrs. --xattrs will enable true and if --xattrs is not added as parameter, it should assign false by default.